guake/guake_app.py

```diff
@@ -101,14 +101,25 @@ class Guake(SimpleGladeApp):
     def __init__(self):
         def load_schema():
             log.info("Loading Gnome schema from: %s", SCHEMA_DIR)
+            with open(os.path.join(SCHEMA_DIR, "version"), encoding="utf-8") as version:
```

Comment:
Prefer to use pathlib (as we only support >= 3.6)

```
with open(pathlib.Path(SCHEMA_DIR / "version"), ...)
```

Reply:
Wasn't os.path.join already around before 3.6? I'll change it to pathlib anyways

Reply:
yep, but pathlib will handle any annoying file system stuff (although Guake didn't run on different OS...)

Reply:
Changed to pathlib